Client/Frontend/Home/Wallpapers/v2/NetworkServices/WallpaperDataService.swift

```diff
@@ -0,0 +1,53 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+
+///  Responsible for fetching data from the server.
+class WallpaperDataService: Loggable {
```

Comment:
Should this have a protocol?

Reply:
No. It used to, but then I switched to properly throwing errors, so this no longer needs it. Will fix. tyty